src/components/aside/Aside.js

```diff
@@ -1,7 +1,54 @@
 import React from 'react';
+import { useEffect, useState } from 'react';
+// import { type } from 'sass';
+import { Link, useNavigate } from 'react-router-dom';
+import './Aside.scss';
 
 const Aside = () => {
-  return <div>무지막지</div>;
+  const [categoryList, setCategoryList] = useState([]);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    fetch('/data/categorylist.json')
+      .then(res => res.json())
+      .then(data => {
+        setCategoryList(data.side);
+      });
+  }, []);
+
+  const productList = type_id => {
+    const queryString = `products/categories?type_id=${type_id}`;
+    navigate(queryString);
+  };
+
+  return (
+    <div className="aside">
+      {categoryList?.map(category => {
+        return (
+          <div className="asideCategory" key={category.category_id}>
+            <Link to={`/products/categories/${category.category_id}/types`}>
+              <h2 className="categoryName">{category.category_name}</h2>
+            </Link>{' '}
```

Comment:
```suggestion
            </Link>
```
{' '} 이부분 필요한 부분인가요?
아마 잘못 들어간 것 같네요!!

항상 push 하시고 깃헙 파일체인지 탭에서 꼼꼼히 확인해주세요!